test/e2e/feature/feature.go

```diff
@@ -120,6 +120,7 @@ var (
 	UDP                                     = framework.WithFeature(framework.ValidFeatures.Add("UDP"))
 	Upgrade                                 = framework.WithFeature(framework.ValidFeatures.Add("Upgrade"))
 	UserNamespacesSupport                   = framework.WithFeature(framework.ValidFeatures.Add("UserNamespacesSupport"))
+	UserNamespacesPodSecurityStandards      = framework.WithFeature(framework.ValidFeatures.Add("UserNamespacesPodSecurityStandards"))
```

Comment:
Don't we need user namespaces support too?

Reply:
I have `[Feature:UserNamespacesSupport] [Feature:UserNamespacesPodSecurityStandards]` in the tests, is that sufficient?

Reply:
I think it might be, but I'm not sure tbh